apps/prairielearn/src/pages/home/home.ejs

```diff
@@ -2,92 +2,64 @@
-              <a href="<%= plainUrlPrefix %>/login" class="btn btn-lg btn-primary">
-                <i class="fa fa-sign-in-alt" aria-hidden="true"></i>
-                Log in
-              </a>
-            </div> <!-- col -->
-
-            <div class="<%= isAuthenticated ? 'col-12' : 'col-md-6 col-lg-5' %>">
-              <div class="row">
-                <div class="<%= isAuthenticated ? 'col-md-6' : 'col-12' %>">
-                  <div class="card rounded-pill my-1">
-                    <div class="card-body d-flex align-items-center p-2">
-                      <span class="fa-stack <%= isAuthenticated ? 'fa-1x mr-1' : 'fa-2x mr-2' %>" aria-hidden="true">
-                        <i class="fas fa-circle fa-stack-2x text-secondary"></i>
-                        <i class="fas fa-user-graduate fa-stack-1x text-light"></i>
-                      </span>
-                      <span>
-                        <p class="<%= isAuthenticated ? 'h6 d-inline-block my-0 ml-n1' : 'h6' %>">
-                          <% if (isAuthenticated) { %>
-                          <span class="btn btn-xs my-0 align-text-bottom btn-white justify-content-center font-weight-bold text-uppercase disabled">
-                            Students
-                          </span>
-                          <% } else { %>
-                          Are you a student?
-                          <% } %>
-                        </p>
-                        <a href="<%= plainUrlPrefix %>/enroll" class="btn btn-<%= isAuthenticated ? 'xs my-0 align-text-bottom' : 'sm mb-1' %> btn-outline-primary justify-content-center">
-                          Enroll course
-                        </a>
-                      </span>
-                    </div><!-- card-body -->
-                  </div> <!-- card -->
-                </div><!-- col -->
-                <div class="<%= isAuthenticated ? 'col-md-6' : 'col-12' %>">
-                  <div class="card rounded-pill my-1">
-                    <div class="card-body d-flex align-items-center p-2">
-                      <span class="fa-stack <%= isAuthenticated ? 'fa-1x mr-1' : 'fa-2x mr-2' %>" aria-hidden="true">
-                        <i class="fas fa-circle fa-stack-2x text-secondary"></i>
-                        <i class="fas fa-user-tie fa-stack-1x text-light"></i>
+          <div class="row">
+            <div class="col-md-6">
+              <div class="card rounded-pill my-1">
+                <div class="card-body d-flex align-items-center p-2">
+                  <span class="fa-stack fa-1x mr-1" aria-hidden="true">
+                    <i class="fas fa-circle fa-stack-2x text-secondary"></i>
+                    <i class="fas fa-user-graduate fa-stack-1x text-light"></i>
+                  </span>
+                  <span>
+                    <p class="h6 d-inline-block my-0 ml-n1">
+                      <span class="btn btn-xs my-0 align-text-bottom font-weight-bold text-uppercase disabled">
```

Comment:
It's weird to see `btn` and `disabled` being used in this context, but I haven't changed it to keep the visual the same. I have removed `justify-content-center` since it has no effect in the context of the parent `d-inline-block`, and `btn-white` which doesn't exist.

Reply:
Yeah, this is kind of strange, and this trips things up a bit in BS5 if I remember right. Might be worth changing this (in a future PR).